microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -160,16 +313,34 @@ See also:
 
 Use this API to check the status of package upload.
 
+##### [v1.1](#tab/v1-1)
+
+<!-- todo: update for 1.1 -->
```

Comment:
h2 #2: update 2 code blocks
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#checking-the-status-of-a-package-upload
@yvsubhash

Reply:
v1.1 code in tabs is ready for review.  closing.